new/detector/composition/javascript/testdata/testcases/class/constructor.js

```diff
@@ -0,0 +1,3 @@
+class User {
+	constructor(name, city) {}
```

Comment:
Should probably support
```javascript
function Person () {
    this.name = 'Sam'
}
```

Reply:
Nice catch.
Would wait for @didroe  on this one as I don't know how to write rule which triggers on `function` node,
I had troubles with this tests case, same troubles we debugged for class. method nodes never get triggered by our framework. 
You'll notice in snapshot that there is no property name or property city on user object, even though tree sitter query should work.